### ISSUE TITLE
multiple processors to structlog.stdlib.ProcessorFormatter

### ISSUE THREAD

**Issue by Akuli** [hynek/structlog#334]
I want structlog messages and stdlib logging to go to stdout and json file. I want timestamps in json file, but not in stdout.

Currently I have to do this (based on [this part of the docs](https://www.structlog.org/en/stable/standard-library.html#rendering-using-structlog-based-formatters-within-logging)):

```python
add_timestamp = structlog.processors.TimeStamper(fmt="%Y-%m-%d %H:%M:%S")
render_json = structlog.processors.JSONRenderer()

def add_timestamp_and_render_json(logger, name, event_dict):
    event_dict = add_timestamp(logger, name, event_dict)
    event_dict = render_json(logger, name, event_dict)
    return event_dict

logging.config.dictConfig({
    ...
    "formatters": {
        "json_formatter": {
            "()": structlog.stdlib.ProcessorFormatter,
            "foreign_pre_chain": [
                structlog.stdlib.add_logger_name,
                structlog.stdlib.add_log_level,
            ],
            "processor": add_timestamp_and_render_json,
        },
        "stdout_formatter": {
            "()": structlog.stdlib.ProcessorFormatter,
            "foreign_pre_chain": [
                structlog.stdlib.add_logger_name,
                structlog.stdlib.add_log_level,
            ],
            "processor": structlog.dev.ConsoleRenderer(),
        },
    },
    ...
})
```

Would be nice if I could provide a chain instead of just one `processor`:

```python
        "json_formatter": {
            ...
            "processor_chain": [
                structlog.processors.TimeStamper(fmt="%Y-%m-%d %H:%M:%S"),
                structlog.processors.JSONRenderer(),
            ]
        },
```

**Comment by hynek** [comment]
Have you tried https://pypi.org/project/multistructlog/ ?

**Comment by Akuli** [comment]
Yes, I have considered using multistructlog. I looked at its source code, and it's very similar to my current structlog config. It doesn't seem to have any way to give a timestamp for one `ProcessorFormatter` but not the other.

**Comment by hynek** [comment]
fixed by #365
